cli/run.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/alibaba/pouch/pkg/reference"
```

Comment:
Please add blank line here.
> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

In https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md

Reply:
ok, i will ready the code_styles.md  carefully